blog/electron-36-0.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Electron 36.0.0
+date: 2025-04-28T00:00:00.000Z
+authors:
+  - georgexu99
+slug: electron-36-0
+tags: [release]
+---
+
+Electron 36.0.0 has been released! It includes upgrades to Chromium 136, V8 13.6, and Node 22.14.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 36.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Bluesky](https://bsky.app/profile/electronjs.org) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Native macOS Context Menu Support
+
+Electron 36 enhances the `menu.popup` API by extending the `options` object to include `frame`. This feature enables integration with macOS system-level features like Writing Tools (spelling and grammar), Autofill, and Services menu items.
+
+```ts
+import { BrowserWindow, Menu, WebFrameMain } from 'electron';
+
+const currentWindow: Electron.BrowserWindow = BrowserWindow.getFocusedWindow();
+const focusedFrame = currentWindow.webContents.focusedFrame;
+const menu = Menu.buildFromTemplate([{ label: 'Copy', role: 'copy' }]);
+
+menu.popup({
+  window: currentWindow,
+  frame: focusedFrame,
+});
+```
+
+### Stack Changes
+
+- Chromium `136.0.7103.48`
+  - [New in 136](https://developer.chrome.com/blog/new-in-chrome-136/)
+  - [New in 135](https://developer.chrome.com/blog/new-in-chrome-135/)
+- Node `22.14.0`
+  - [Node 22.14.0 blog post](https://nodejs.org/en/blog/release/v22.14.0/)
+- V8 `13.5`
+
+Electron 36 upgrades Chromium from `134.0.6998.23` to `136.0.7103.48`, and V8 from `13.5` to `13.6`.
+
+### New Features
+
+- Added `BrowserWindow.isSnapped()` to indicate whether a given window has been arranged via Snap. [#46226](https://github.com/electron/electron/pull/46226)
+- Added `WebContents.focusedFrame` to get the focused frame.
+  - Fixed `WebContents.opener` to specify potential `null` type. [#45667](https://github.com/electron/electron/pull/45667)
+- Added `ffmpeg.dll` to delay load configuration. [#46173](https://github.com/electron/electron/pull/46173) <span style="font-size:small;">(Also in [34](https://github.com/electron/electron/pull/46174), [35](https://github.com/electron/electron/pull/46172))</span>
```

Comment:
```suggestion
- Added `ffmpeg.dll` to delay load configuration. [#46173](https://github.com/electron/electron/pull/46173) <span style="font-size:small;">(Also in [34](https://github.com/electron/electron/pull/46174), [35](https://github.com/electron/electron/pull/46172))</span>
```

The build is failing because we need to use the `style={{}}` prop in the way that JSX expects it.